spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -319,11 +378,12 @@ public DistributionMode positionDeltaMergeDistributionMode() {
     return DistributionMode.fromName(mergeModeName);
   }
 
-  public boolean useTableDistributionAndOrdering() {
+  private boolean ignoreTableDistributionAndOrdering() {
     return confParser
         .booleanConf()
         .option(SparkWriteOptions.USE_TABLE_DISTRIBUTION_AND_ORDERING)
         .defaultValue(SparkWriteOptions.USE_TABLE_DISTRIBUTION_AND_ORDERING_DEFAULT)
+        .negate()
```

Comment:
This makes me slightly apprehensive, since I don't like flipping the sign in the builder. I do think it's cleaner than just adding a ! to the return statement though so +1

Reply:
I debated it but having negation in the main logic bothered me more :)